### PR TITLE
Fix polyfills and default app redirection

### PR DESCRIPTION
**Pull request by CPatchane**

### CODE REVIEW

File: src/application.coffee [hunk 1]
```diff
@@ -82,15 +62,8 @@ class App extends Application
     # Redirect to given app
     handleTriggerDone: () ->
         url = "#{window.location.protocol}//#{@domain}"
```
Comment (nono) [inline src/application.coffee]: `url = "//#{@domain}"` should be enough

**Review by nono — APPROVED** [review]

File: src/application.coffee (hunk 1 above)
Comment (CPatchane) [reply src/application.coffee]: Right, I'll fix it. Thanks 👍

File: src/lib/onboarding.coffee [hunk 2]
```diff
@@ -271,12 +271,10 @@ module.exports = class Onboarding
             instance = JSON.parse window.localStorage.getItem 'instance'
         catch e
             instance = null
-        console.debug instance
```
Comment (gregorylegarec) [inline src/lib/onboarding.coffee:274]: ❤️

File: src/lib/onboarding.coffee [hunk 3]
```diff
@@ -271,12 +271,10 @@ module.exports = class Onboarding
...
         return instance
 
 
     saveInstanceLocally: (instance) ->
-        console.debug instance
```
Comment (gregorylegarec) [inline src/lib/onboarding.coffee:279]: 😍

**Review by gregorylegarec — APPROVED** [review]
👍

**Comment by gregorylegarec** [comment]
Thanks @CPatchane, I hope we will be in peace with polyfills one day.

**Pull request merged**